server/mldb_server.cc

```diff
@@ -499,8 +499,8 @@ getPackageDocumentationPath(const Package & package) const
     // by any means guaranteed.
 
     if (package.packageName() == "builtin")
-        return "/doc/builtin/";
-    return "/v1/plugins/" + package.packageName() + "/doc/";
+        return httpBaseUrl + "/doc/builtin/";
+    return httpBaseUrl + "/v1/plugins/" + package.packageName() + "/doc/";
```

Comment:
Personally, I don't think we should use the boost::filesystem for URLs, though I'd be curious to understand the advantages of b::f::operator/